# origin: Debian (bcollins)
# cset: n/a
# inclusion: not suitable for upstream
# revision date: 2004-10-08

Seemingly only on the onboard hme's on the Ultra 1E (enterprise) models
(as the non-E models have an onboard le)... it's a strange lockup problem
that requires a reboot to fix.  This one-liner is a band-aid that seems
to take care of it... a possible weirdo timing bug. --Uzi

--- kernel-source-2.6.11-2.6.11-orig/drivers/net/sunhme.c	2005-03-02 08:37:55.000000000 +0100
+++ kernel-source-2.6.11-2.6.11/drivers/net/sunhme.c	2005-03-05 10:48:16.704927336 +0100
@@ -1990,6 +1990,7 @@
 	}
 	hp->tx_old = elem;
 	TXD((">"));
+	udelay(1);
 
 	if (netif_queue_stopped(dev) &&
 	    TX_BUFFS_AVAIL(hp) > (MAX_SKB_FRAGS + 1))
